gate-core/src/main/java/com/netflix/spinnaker/gate/config/Service.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ * Copyright 2023 Apple, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.config;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.StringUtils;
+
+@Getter
+@Setter
+public class Service {
+  private boolean enabled = true;
+  private String baseUrl;
+  private MultiBaseUrl shards;
```

Comment:
The groovy code has
```
int priority = 1
```

Reply:
Huh that one I think is used... in the sharding system i THINK...

Reply:
Unused variable as well. The priority of individual `MultiBaseUrl` objects is taken into account, but a general `Service` entry does not (probably because they're keyed by service name).